config/backend-config/namespace_config_test.go

```diff
@@ -99,6 +99,12 @@ func Test_Namespace_Get(t *testing.T) {
 		)
 	}
 
+	t.Log("correct workspaceID to dataRetention settings mapping")
+	w1DataRetention := DataRetention{EnableReportingPii: true}
+	w2DataRetention := DataRetention{EnableReportingPii: false}
+	require.Equal(t, w1DataRetention, client.GetDataRetentionSettingsForWorkspaceID(workspaceID1))
+	require.Equal(t, w2DataRetention, client.GetDataRetentionSettingsForWorkspaceID(workspaceID2))
```

Comment:
Given your implementation this would also be true:

```go
require.Equal(t, w2DataRetention, client.GetDataRetentionSettingsForWorkspaceID("NON-EXISTENT"))
```

Again, it might not be a problem after all, just pointing it out.

Reply:
Added this check also.